modules/accredited_representative_portal/app/models/accredited_representative_portal/power_of_attorney_holder.rb

```diff
@@ -0,0 +1,93 @@
+# frozen_string_literal: true
+
+# rubocop:disable Metrics/MethodLength
+module AccreditedRepresentativePortal
+  class PowerOfAttorneyHolder <
+    Data.define(
+      :type,
+      :poa_code,
+      :can_accept_digital_poa_requests
+    )
+
+    module Types
+      ALL = [
+        ##
+        # Future types:
+        # ```
+        # ATTORNEY = 'attorney',
+        # CLAIMS_AGENT = 'claims_agent',
+        # ```
+        #
+        VETERAN_SERVICE_ORGANIZATION = 'veteran_service_organization'
+      ].freeze
+    end
+
+    PRIMARY_KEY_ATTRIBUTE_NAMES = %i[
+      type
+      poa_code
+    ].freeze
+
+    class << self
+      # The `for_user` method retrieves the Power of Attorney (POA) holders associated with a user
+      # based on their email and ICN (Integration Control Number).
+      #
+      # Why this method exists:
+      # - The purpose of this method is to determine which organizations or individuals hold
+      #   power of attorney for a given user.
```

Comment:
@ojbucao Our users are representatives. This determines which VSOs the representative belongs to.